### ISSUE TITLE
Absence of Validation set?

### ISSUE THREAD

**Issue by kkontras** [GeWu-Lab/OGM-GE_CVPR2022#43]
Hi,

I was trying to run the methods based on your code and I realized that you split the dataset based on the given training and test set but there was no further split for validation set. I wanted to ask whether you have tuned on the test set and if yes how did it come up?
